documentation/explainers/synthetic_tokens.md

```diff
@@ -0,0 +1,88 @@
+# Overview
+
+Synthetic tokens are collateral-backed tokens whose value fluctuates depending on the tokens’ reference index. Synthetic tokens blend features of prediction markets, futures markets, and collateralized loans. 
+
+Some examples of synthetic tokens include: 
+- synthetic real-world assets
```

Comment:
do you want to give an example here in brackets?
```suggestion
- synthetic real-world assets (eg: gold or Tesla stock price)
```

Reply:
I think it may be better to leave them without examples for now (or potentially link to NFP/things like US Stocks) in the future.